CHANGES.md

```diff
@@ -1,3 +1,19 @@
+## edge-19.8.7
+
+* CLI
+  * Added a global `--cluster-domain` flag to `linkerd install` to allow
+    installing Linkerd into a Kubernetes cluster that uses a base domain other
+    than `cluster.local.` (thanks @arminbuerkle!)
+* Web UI
+  * Fixed an issue that caused unnecessary Prometheus queries, reducing load on
```

Comment:
The web UI change was an improvement/refactoring, not a bug fix -- I would say "Increased dashboard speed by consolidating existing Prometheus queries", would defer to @alpeb

Reply:
Correct, it's an improvement (not huge though, just 4% less queries). I would phrase it like "Reduce Prometheus query load when using the dashboard"